guides/source/action_view_overview.md

```diff
@@ -1230,7 +1230,7 @@ Creates a field set for grouping HTML form elements.
 Creates a file upload field.
 
 ```html+erb
-<%= form_tag({ action: "post" }, multipart: true) do %>
+<%= form_with url: { action: "post" }, multipart: true do %>
```

Comment:
Maybe we should change this to `url: new_account_avatar_path(@account)` and rename "file" to avatar?

If we do `form.file_field` do we need the `multipart: true`?